internal/envoy/install.go

```diff
@@ -32,6 +32,7 @@ import (
 var binEnvoy = filepath.Join("bin", "envoy"+moreos.Exe)
 
 // InstallIfNeeded downloads an Envoy binary corresponding to globals.GlobalOpts and returns a path to it or an error.
+//nolint
```

Comment:
we don't want function-scoped `//nolint` in main code even if it could be ok in tests. Probably better to not use this scope in general as it subverts the linter for all code enclosed.

Reply:
for example, golint adds linters on almost all releases, so in the future this may prevent us from knowing a problem there's no linter now for.